### DISCUSSION TITLE
Why default AdGuard Home setup uses Quad9 vs AdGuard itself?

### DISCUSSION THREAD

**Discussion by logicito** [AdguardTeam/AdGuardHome#7161] [General]
I installed AdGuard Home in my OpenWrt router, and noticed that the Upstream DNS server is Quad9 (https://dns10.quad9.net/dns-query) only, not even the AdGuard servers, is there is reason for this?
Should I add more DNS servers as backup? Thanks

**Comment by ainar-g** [comment]
As far as I know, at the moment when AdGuard Home was still in early development, AdGuard DNS wasn't yet ready for its prime-time.  We may switch to AdGuard DNS Unfiltered in the future, though.
